### PR TITLE
improve error handling

### PR DESCRIPTION
**Pull request by RamonSaboya**
As it used to be, whenever an error occurred, the whole application would get restarted. With this PR, it will only fail the current request.

### CODE REVIEW

File: api/main.go [hunk 1]
```diff
@@ -42,11 +59,17 @@ func templatesHandler(w http.ResponseWriter, r *http.Request) {
 func requestCvHandler(w http.ResponseWriter, r *http.Request, ch *amqp.Channel, q amqp.Queue) {
 	body, err := ioutil.ReadAll(r.Body)
 	defer r.Body.Close()
-	failOnError(err, "Failed to parse body")
+	if err != nil {
+		http.Error(w, errMsg(err, "Failed to parse body"), http.StatusInternalServerError)
```
Comment (Arthurlpgc) [inline api/main.go]: Are we sending the errors to logger

File: storage/main.go [hunk 2]
```diff
@@ -10,9 +11,13 @@ import (
 	"github.com/rs/cors"
 )
 
+func errMsg(err error, msg string) string {
```
Comment (Arthurlpgc) [inline storage/main.go]: 3 copies of same code, 1 more and we need a lib

Comment (RamonSaboya) [reply storage/main.go]: overkill for now IMO. But let's keep an eye on it

File: api/main.go (hunk 1 above)
Comment (RamonSaboya) [reply api/main.go]: check new commit

**Pull request merged**